### ISSUE TITLE
[FEATURE] Mark a task as down easily

### ISSUE THREAD

**Issue by linsui** [ahmet-cetinkaya/whph#90]
### Is your feature request related to a problem?

When I see a notification of a task, I need to open whph and click the small circle. This is a little difficult. Sometimes I need to try a few times to mark it as done.

### Describe the solution you'd like

Allow mark a task as done from the notification directly and allow sweap to mark a task as done.

### Feature Category

Task Management

**Comment by ahmet-cetinkaya** [comment]
Thanks @linsui!

You're right - it should be easier to mark tasks done from notifications.

I'll add:
- Notification action button to mark tasks done directly
- Larger touch targets (Material Design standard)

This should make it much smoother on mobile. I'll work on this soon.

**Comment by linsui** [comment]
Thanks!

**Comment by linsui** [comment]
Thanks!
